index.html

```diff
@@ -39,6 +42,8 @@
         document.getElementById('sel-themes').addEventListener('change', function () {
             cssLink.href = 'bower_components/medium-editor/dist/css/' + this.value + '.css';
         });
+
+        document.getElementById('version').innerHTML = 'This demo uses medium-editor v'+MediumEditor.version.toString();
```

Comment:
Yep, I first would like to display without casting, so I let the `.toString()` after :)